scripts/addFromConfig.py

```diff
@@ -322,6 +317,20 @@ def addStuff(cfg, options):
         # recheck the temp file
         conf = dbprocessing.Utils.readconfig(tmpf.name)
         configCheck(conf)
+        ### The following is commented out as a Postgres database is implemented.
+        # do all our work on a temp version of the DB, if it all works, move tmp on top of existing
+        #   if it fails just delete the tmp and do nothing
+        # orig_db = options.mission
+        # tmp_db = tempfile.NamedTemporaryFile(delete=False, suffix='_temp_db')
+        # tmp_db.close()
+        # shutil.copy(orig_db, tmp_db.name)
+        # options.mission = tmp_db.name
+        try:
+            addStuff(conf, options)
+            # shutil.copy(tmp_db.name, orig_db)
+        finally:
+            pass
+            # os.remove(tmp_db.name)
         if os.path.isfile(options.mission): # sqlite
             # do all our work on a temp version of the DB, if it all works, move tmp on top of existing
             #   if it fails just delete the tmp and do nothing
```

Comment:
This all got really mixed up because it for some reason ripped out the code which handled both sqlite and postgres, replaced it with postgres-only, and then added the original back in (throughout the merges and such). Any reason not to just keep with the current master branch, which does both?

Reply:
That does not work, because your mission is a filename and mine is not.  I struggled a lot with your temporarily file and shutil.copy.   I had to use this code instead whihc worked on my linux box. 
```
        tmpf = tempfile.NamedTemporaryFile(mode='wt', delete=False,
                                           suffix='_conf_file')
        tmpf.file.writelines(cfg)
        tmpf.close()
        # recheck the temp file
        conf = dbprocessing.Utils.readconfig(tmpf.name)
        configCheck(conf)
```

Reply:
Right, but as it was in current master, it checked for whether the mission is a filename and skips all the tempfile stuff.

Reply:
I'm going to revert this in favor of the version on master, which checks if the mission is a file and does the handling differently based on whether it's a file (treat like sqlite) or not (treat as postgres). What's posted in the comment above (tempf handling) is identical between master and the PR.